terraform/shared/video-player/common.js

```diff
@@ -0,0 +1,17 @@
+// JavaScript Document
+ipfs_gateway_self='__IPFS_GATEWAY_SELF__';  // IPFS gateway of this node
+ipfs_gateway_origin='__IPFS_GATEWAY_ORIGIN__';  // IPFS gateway of origin stream
+m3u8_ipfs='__IPFS_GATEWAY_ORIGIN__/ipns/__IPFS_ID_ORIGIN__';  // URL to m3u8 over IPFS gateway
+m3u8_http='__HTTP_M3U8__';  // Optional URL to m3u8 over HTTP

```

Comment:
Needs to be an array, this is how @ASoTNetworks  does "CDN" by randomly picking one of the gateways to stream from.

Reply:
`m3u8_http=['server1','sever2','server3'];`